types/lodash/lodash-tests.ts

```diff
@@ -11266,6 +11266,22 @@ namespace TestGet {
     _.get([], Symbol.iterator);
     _.get([], [Symbol.iterator]);
 
+    _.get('abc', 4); // $ExpectType string
+    _('abc').get(4); // $ExpectType string
+    _.chain('abc').get(4); // $ExpectType LoDashExplicitWrapper<string>
+
+    _.get({ a: { b: true } }, "a"); // $ExpectType { b: boolean; }
```

Comment:
How are these following tests related to the addition of NumericDictionary overloads? I didn't think object literals had numeric index signatures, and `"a"` isn't of type number, either.

Reply:
Yeah, they're not related to `NumericDictionary`. I added them because it seemed that we were lacking tests for the `object` overloads.